docs/algorithms.md

```diff
@@ -529,32 +515,37 @@ cd ./cocos
 Start the computation server:
 
 ```bash
-go run ./test/computations/main.go ./test/manual/algo/lin_reg.py public.pem false ./test/manual/data/iris.csv
+go run ./test/computations/main.go ./test/manual/algo/linreg.tar public.pem false ./test/manual/data/iris.csv
 ```
 
 Start the manager
 
+```bash
+cd cmd/manager
+```
+
 ```bash
 sudo \
 MANAGER_QEMU_SMP_MAXCPUS=4 \
+MANAGER_QEMU_MEMORY_SIZE=25G \
 MANAGER_GRPC_URL=localhost:7001 \
 MANAGER_LOG_LEVEL=debug \
 MANAGER_QEMU_ENABLE_SEV_SNP=false \
 MANAGER_QEMU_OVMF_CODE_FILE=/usr/share/edk2/x64/OVMF_CODE.fd \
 MANAGER_QEMU_OVMF_VARS_FILE=/usr/share/edk2/x64/OVMF_VARS.fd \
-go run ./cmd/manager/main.go
+go run main.go
 ```
 
 Export the agent grpc url from computation server logs
 
 ```bash
-export AGENT_GRPC_URL=localhost:6015
+export AGENT_GRPC_URL=localhost:6100
```

Comment:
Add an indication of what would happen if the port is not avaiable. Or indicate why start with 6100.

Reply:
I don't think this should be covered here since we don't cover edge cases